### PR TITLE
Code Improvements

### PR DESCRIPTION
**Pull request by darxriggs**

### CODE REVIEW

**Comment by darxriggs** [comment]
Hi @casz, can you have a look at this?

**Comment by casz** [comment]
😎 There @darxriggs now you got some merge conflict to deal with 😇

File: checkstyle.xml [hunk 1]
```diff
@@ -3,6 +3,10 @@
 <module name="Checker">
     <module name="FileTabCharacter" />
     <module name="NewlineAtEndOfFile" />
+    <module name="RegexpMultiline">
```
Comment (casz) [inline checkstyle.xml]: Please do not use this module 😅
The proper way is `.gitattributes` with `git add --renormalize .` 😇

Comment (casz) [reply checkstyle.xml]: See https://github.com/jenkinsci/vstestrunner-plugin/pull/16 for reference

Comment (darxriggs) [reply checkstyle.xml]: I am not a big fan of automatic conversions. But I implemented your suggestion.

**Comment by darxriggs** [comment]
Hi @casz, now I am done.

**Pull request merged**